cms/migrations/0037_learning_outcomes_subhead_richtext.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.1.9 on 2019-07-03 09:59
+import re
+import wagtail.core.fields
+from django.db import migrations
+from django.db.models import F, Value
+from django.db.models.functions import Concat
+
+
+def wrap_in_paragraph(apps, schema_editor):
+    """
+    Wraps the CharField value with paragraph tags so the subsequent migration
+    to convert this field into a RichTextField will not impact any visual
+    display on client end.
+    """
+    if not schema_editor.connection.alias == "default":
+        return
+    LearningOutcomesPage = apps.get_model("cms", "LearningOutcomesPage")
+    LearningOutcomesPage.objects.update(
+        sub_heading=Concat(Value("<p>"), F("sub_heading"), Value("</p>"))
```

Comment:
I know that Wagtail wraps your text in a `<p>` tag if you edit a rich text field value, but do we _need_ to backpopulate that? What happens if you just leave these values alone?